engine/classes/Elgg/Application.php

```diff
@@ -59,6 +54,7 @@ class Application {
 	 */
 	private static $public_services = [
 		//'config' => true,
+		'session' => true,
```

Comment:
We already have elgg_get_session(). And aren't we trying to get rid of these?

Reply:
Yes, need to remove